opennlp-tools/src/main/java/opennlp/tools/util/PlainTextByLineStream.java

```diff
@@ -59,19 +55,12 @@ public void reset() throws IOException {
     if (inputStreamFactory != null) {
```

Comment:
Is this `if` needed since there is no `else` for it?

Reply:
I think the original idea of this check was to select between various ways of passing the datasource (File/Inputstream) into the class.  Now that we pass in the factory, I don't think we need to worry about this.  Unless, the user passes in a null factory.  In that case, it is appropriate to throw a NullPointerException.